contrib/storage-mongo/src/main/java/org/apache/drill/exec/store/mongo/MongoGroupScan.java

```diff
@@ -180,39 +179,39 @@ private void init() throws IOException {
     chunksMapping = Maps.newHashMap();
     chunksInverseMapping = Maps.newLinkedHashMap();
     if (isShardedCluster(client)) {
-      MongoDatabase db = client.getDatabase(CONFIG);
-      MongoCollection<Document> chunksCollection = db.getCollection(CHUNKS);
+      MongoDatabase db = client.getDatabase(DrillMongoConstants.CONFIG);
```

Comment:
Whole point here was to avoid using an interface for constants and to avoid doing a static/wildcard import, following [Google guidelines](
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports). However, if this is taken as a show stopper, I can switch to static imports, disliking myself later ;)